cmd/monaco/integrationtest/account/utils_test.go

```diff
@@ -23,49 +23,10 @@ import (
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/files"
 	"github.com/spf13/afero"
 	"github.com/stretchr/testify/require"
-	"regexp"
 	"strings"
 	"testing"
 )
 
-type monacoCmd struct {
-	cmd string
```

Comment:
could we just use a slice of strings here rather than splitting them later?

Reply:
It can be changed, it is internal implementation